examples/llm_pte_finetuning/runner.py

```diff
@@ -30,6 +30,18 @@
 )
 parser.add_argument("--cfg", type=str, help="Path to the config file.")
 parser.add_argument("--model_file", type=str, help="Path to the ET model file.")
+parser.add_argument(
+    "--num_training_steps",
+    type=int,
+    help="Number of training steps, assuming 1 epoch.",
+    default=1,
```

Comment:
should we do a different default here? Like the avg number we've noticed to see some convergence?

Reply:
I updated it to 5 eval steps and 100 steps as we did in our manual tests